lib/models/movie_model.dart

```diff
@@ -0,0 +1,59 @@
+import 'movie.dart';
+
+import 'genre_model.dart';
+
+class MovieModel {
```

Comment:
This class is not necessary at all, you have the Movie class created and also you have the GenresModel that has a List<Genres>

Reply:
makes sense, fixed